scripts/drone/utils/images.star

```diff
@@ -8,9 +8,10 @@ load(
     "nodejs_version",
 )
 
+# "go" image can be switched back to golang:{}-alpine once this is resolved https://github.com/mattn/go-sqlite3/pull/1177#issuecomment-1849176090
 images = {
     "git": "alpine/git:2.40.1",
-    "go": "golang:{}-alpine".format(golang_version),
+    "go": "golang:{}-alpine3.18".format(golang_version),
```

Comment:
trying another image to avoid https://github.com/mattn/go-sqlite3/pull/1177#issuecomment-1849176090